script.js

```diff
@@ -1,7 +1,9 @@
-// Book object: Represents a Book
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
+// Book Class: Represents a Book
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
```

Comment:
- [ ] Great job so far but I think you forget to add the methods inside the class as required since the refactor it should add all the method inside the class with keeping the functionality so please fix it and refactor it to have the methods inside the class as required 🔥🤩⬇💯💯


![image](https://user-images.githubusercontent.com/58553711/235911778-d3f53169-4c2e-4704-ada4-3b6ba43d69d3.png)